.ci/lib.sh

```diff
@@ -358,27 +358,30 @@ gen_clean_arch() {
 		info "Remove installed containerd-cri related binaries and configuration"
 		delete_containerd_cri_stale_resource
 	fi
-
-	if [[ $CI_JOB != "METRICS" ]]; then
-		#reset k8s service may impact metrics test on x86_64, so limit it to arm64
-		[ $(uname -m) == "aarch64" -a "$(pgrep kubelet)" != "" ] && sudo sh -c 'kubeadm reset -f'
+		# reset k8s service may impact metrics test on x86_64
```

Comment:
Nit: The indentation in this block is too deep (one too many tabs fwics).

Reply:
@jodh-intel thank you, done